src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
```

Comment:
next time try to keep the original method signature, since most of the time it defines a way how a task should have been done in the first place